vars/checksPublishResults.groovy

```diff
@@ -0,0 +1,240 @@
+import com.cloudbees.groovy.cps.NonCPS
+
+import com.sap.piper.ConfigurationLoader
+import com.sap.piper.ConfigurationMerger
+
+import groovy.transform.Field
+
+@Field def STEP_NAME = 'checksPublishResults'
+
+/**
+ * checksPublishResults
+ *
+ * @param others document all parameters
+ */
+def call(Map parameters = [:]) {
+    handlePipelineStepErrors (stepName: STEP_NAME, stepParameters: parameters) {
+        def script = parameters.script
+        if (script == null)
+            script = [commonPipelineEnvironment: commonPipelineEnvironment]
+        prepareDefaultValues script: script
+
+        Map configurationKeys = [
```

Comment:
There is no need to have that inside the call(Map) method. Here is gets instanciated with each method call, ok this is not a big thing, but nevertheless it is not needed. With the ```@Field``` annotation we could provide this outside. Makes IMO the call method clearer since this is less noisy (we should change that IMO also in the other steps)

Reply:
currently the parameter / step config key map is the same (at least the keys) as the default config map. Should we use this here instead?